geolocator/lib/geolocator.dart

```diff
@@ -261,15 +261,43 @@ class Geolocator {
         timeLimit: timeLimit,
       );
 
-  /// Fires whenever the location changes inside the bounds of the
+  /// Returns a stream emitting NMEA-0183 sentences when they are received from
+  /// the GNSS engine. With devices running a Android API level lower than 24
+  /// NMEA-0183 sentences are received from the GPS engine.
```

Comment:
nit: the explanation is very clean but Dart documentation guidelines prescribe the first sentence to be short and compact. So maybe we can update this to:

```suggestion
  /// Returns a stream emitting NMEA-0183 sentences (Android only, no-op on iOS). 
  ///
  /// With devices running a Android API level lower than 24 NMEA-0183 sentences 
  /// are received from the GPS engine. From API level 24 and up the GNSS engine 
  /// is used.
```